### PR TITLE
Update slam3d

### PR DESCRIPTION
**Pull request by skasperski**
Depends on dfki-ric/slam3d#8

### CODE REVIEW

**Comment by planthaber** [comment]
I don't remember if it was intentional to include the TSDF Map into this PR, but I'm not against it.

File: slam3d.orogen [hunk 1]
```diff
@@ -10,7 +10,7 @@ import_types_from 'base'
 import_types_from 'maps'
 import_types_from 'pcl'
 import_types_from 'slam3d/sensor/pcl/RegistrationParameters.hpp'
-import_types_from 'maps/grid/MLSConfig.hpp'
+#import_types_from 'maps/grid/MLSConfig.hpp'
```
Comment (planthaber) [inline slam3d.orogen]: This could be completely removed

**Review by planthaber — APPROVED** [review]

File: slam3d.orogen (hunk 1 above)
Comment (skasperski) [reply slam3d.orogen]: Done.

**Comment by skasperski** [comment]
> I don't remember if it was intentional to include the TSDF Map into this PR, but I'm not against it.

No it wasn't, thanks for mentioning. I reverted this commit here.

**Pull request merged**